changelogs/21.10

```diff
@@ -0,0 +1,72 @@
+* COSMIC
+** GNOME 40/41 support for COSMIC and Pop Shell.
+** Re-enabling support for vertical workspaces.
+** Added a new Application Library interface to replace the GNOME application overview.
+** Various UX improvements to the launcher have been made across this release.
+** And a handful of issues in Pop Shell have been resolved as well.
+** Emoji picker set back to Ctrl + Shift + E
++* Alt+F4 added as an alternative to Super+Q for closing windows
+
+* Application Library
+** Replaces the GNOME application overview, achieving the same information density with less space.
+** Improves multi-monitor experience by launching onto the display that has mouse focus.
+** Alphabetical application sorting will make it easier to locate the application or folder you want.
+** Drag and drop applications into custom folders which are visible and act as tabs.
+** Search quickly filters installed applications and applications that are available to install from the Pop!_Shop.
+** Achieves the same information density while using much less space.
+** Provides the optimal experience, especially on ultra-wide monitors.
+
+* Pop! Shell
+** Various scenarios causing crashes were resolved.
+** Allow floating window exceptions to be tiled when toggled.
+** Transitions removed on opening a window to fix windows being stretched sometimes.
+** Dialogs will no longer unmaximize a maximized window when they gain focus.
+** Improved high contrast support.
+** Many improvements to the tiling behaviors of many unique window types.
+** Improvements to window attachment behaviors when new windows are spawned.
+
+* Pop! Shop
+** Many PRs have been submitted to elementary OS to address a number of problematic areas we've experienced.
+** The shop is currently in the process of being rebased, and will be released soon.
+** Issues with packagekit and libflatpak causing freezes will be worked on in this release.
+
+* Pop! PPA now hosted by System76
+** A new packaging CI was developed in Rust for speedier package builds.
+** Which enables us to test fixes and improvements faster.
+** Packages are now published and delivered from apt.pop-os.org instead of Launchpad.
+** Which allows us to deliver updates quicker, with more control over package builds.
+
+* Mainline Linux kernel
+** Starting with Linux 5.15, Pop!_OS will be shipping with the mainline kernel with the latest compatible NVIDIA drivers.
+** New kernel policy whereby the latest kernels will be released once they've passed extensive quality assurance tests.
+** System76 hardware lab contains a broad spectrum of chipsets, processors, and components for regression testing.
+
+* Improved System Stability When Installing & Upgrading Software
+** Apt is now prevented from enabling the accidental removal of critical desktop packages.
+** Idea pitched to Debian and Debian issued a similar fix that we now ship.
```

Comment:
Not sure if this line belongs here-- the previous line covers the actual change, this is just part of the story of how we got there.